landingzones/tests/test_permissions_api.py

```diff
@@ -127,3 +132,101 @@ def test_retrieve_archive(self):
         self.assert_response(url, good_users, 200)
         self.assert_response(url, bad_users, 403)
         self.assert_response(url, [self.anonymous], 401)
+
+    def _get_post_data(self):
+        return json.dumps(
+            {
+                'assay': str(self.assay.sodar_uuid),
+                'description': 'Test description updated',
+            }
+        )
+
+    def test_update(self):
+        """Test LandingZoneUpdateAPIView permissions"""
+        url = reverse(
+            'landingzones:api_update',
+            kwargs={'landingzone': self.landing_zone.sodar_uuid},
+        )
+        good_users = [
+            self.superuser,
+            self.user_owner_cat,
+            self.user_owner,
+            self.user_delegate,
+        ]
+        bad_users = [
+            self.user_contributor,
+            self.user_guest,
+            self.user_no_roles,
+        ]
+        self.assert_response_api(
+            url,
+            good_users,
+            200,
+            method='PATCH',
+            data=self._get_post_data(),
+            knox=True,
+            req_kwargs={'CONTENT_TYPE': 'application/json'},
```

Comment:
These content type redefinitions should not be needed. I'm guessing there is something wrong with either the view or test class setup. I'll take a closer look and get back to you on this one.